File: v2-microservicios/python-service/schemas/usuario.py
from utils.ma import ma
from model.usuario import Usuario
from marshmallow import fields

class UsuarioSchema(ma.Schema):
    class Meta:
        model = Usuario
        fields = (
            'id_usuario',
            'email',
            'clave',
            'tipo_usuario',
        )

usuario_schema = UsuarioSchema()
usuarios_schema = UsuarioSchema(many=True)

File: v2-microservicios/sisvita-app/auth-server/src/main/java/sm/dsw/sisvita/auth/model/Usuario.java
package sm.dsw.sisvita.auth.model;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "usuario")
public class Usuario {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id_usuario;
    @Column(name = "email", unique = true)
    private String email;

    @Column(name = "clave")
    private String clave;

    @Column(name = "tipo_usuario")
    private String tipo_usuario;
}
